src/shadowenv.rs

```diff
@@ -34,8 +34,10 @@ impl Shadowenv {
         env: HashMap<String, String>,
         shadowenv_data: undo::Data,
         target_hash: u64,
+        clobber: bool,
     ) -> Shadowenv {
         let (unshadowed_env, no_clobber, prev_dirs) = Shadowenv::unshadow(&env, shadowenv_data);
+        let no_clobber = if clobber { HashSet::new() } else { no_clobber };
```

Comment:
do you use this variable somewhere?

Reply:
yeah it's used down here https://github.com/Shopify/shadowenv/blob/9de412824a7e1133e3d38a3cffd5c7069e42b3ed/src/shadowenv.rs#L45

i should really clean up the naming a bit. this area reads pretty unintuitively.